bundle/render/render_text_output_test.go

```diff
@@ -519,6 +519,10 @@ func TestRenderSummary(t *testing.T) {
 						URL:         "https://url2",
 						JobSettings: &jobs.JobSettings{Name: "job2-name"},
 					},
+					"job3": {
+						ID:  "3",
+						URL: "https://url3", // This emulates deleted job
+					},
```

Comment:
If I understand correctly, this PR would allow such config to be loaded and deployed but will skip over this particular job.

Why don't we reject such jobs at validation stage instead?

Reply:
Not exactly. This is a "bundle summary" command which essentially shows the current state of your bundle configuration. https://github.com/databricks/cli/pull/1123 
It is used in VS Code for rendering the state of resources in the side panel as well as directly by customers to see the state of the bundle.

Reply:
For validation stage we have a mutator which does check that all resources are defined correctly https://github.com/databricks/cli/blob/main/bundle/config/validate/all_resources_have_values.go#L14